sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/src/test/java/com/azure/messaging/eventhubs/checkpointstore/jedis/JedisRedisCheckpointStoreTests.java

```diff
@@ -0,0 +1,172 @@
+            .expectError(IllegalStateException.class)
+            .verify();
+    }
+
+    @Test
+    public void testListOwnership() {
+        PartitionOwnership partitionOwnership = new PartitionOwnership()
+            .setFullyQualifiedNamespace(FULLY_QUALIFIED_NAMESPACE)
+            .setEventHubName(EVENT_HUB_NAME)
+            .setConsumerGroup(CONSUMER_GROUP)
+            .setPartitionId(PARTITION_ID)
+            .setOwnerId("ownerOne")
+            .setETag("eTag");
+
+        Set<String> value = new HashSet<>();
+        value.add(KEY);
+        byte[] bytes = jsonSerializer.serializeToBytes(partitionOwnership);
+        List<String> list = Collections.singletonList(new String(bytes, StandardCharsets.UTF_8));
+
+        when(jedisPool.getResource()).thenReturn(jedis);
+        when(jedis.smembers(PREFIX)).thenReturn(value);
+        when(jedis.hmget(eq(KEY),
+            eq(JedisRedisCheckpointStore.PARTITION_OWNERSHIP))).thenReturn(list);
+
+        StepVerifier.create(store.listOwnership(FULLY_QUALIFIED_NAMESPACE, EVENT_HUB_NAME, CONSUMER_GROUP))
+            .assertNext(partitionOwnershipTest -> {
+                assertEquals(FULLY_QUALIFIED_NAMESPACE, partitionOwnershipTest.getFullyQualifiedNamespace());
+                assertEquals(EVENT_HUB_NAME, partitionOwnershipTest.getEventHubName());
+                assertEquals(CONSUMER_GROUP, partitionOwnershipTest.getConsumerGroup());
+                assertEquals("ownerOne", partitionOwnershipTest.getOwnerId());
+            })
+            .verifyComplete();
+    }
+
+    @Test
+    public void testListOwnershipEmptyList() {
+        when(jedisPool.getResource()).thenReturn(jedis);
+        when(jedis.smembers(PREFIX)).thenReturn(new HashSet<>());
+
+        StepVerifier.create(store.listOwnership(FULLY_QUALIFIED_NAMESPACE, EVENT_HUB_NAME, CONSUMER_GROUP))
+            .expectNextCount(0L)
+            .verifyComplete();
+    }
+
+    @Test
+    public void testClaimOwnership() {
+        //Test will change when claimOwnership is fully implemented
+        List<PartitionOwnership> partitionOwnershipList = new ArrayList<>();
+        StepVerifier.create(store.claimOwnership(partitionOwnershipList))
+            .expectNextCount(0L)
+            .verifyComplete();
+    }
+
+    @Test
+    public void testUpdateCheckpoint() {
+        //Test will change when claimOwnership is fully implemented
+        Checkpoint checkpoint = new Checkpoint()
+            .setConsumerGroup(CONSUMER_GROUP)
+            .setEventHubName(EVENT_HUB_NAME)
+            .setFullyQualifiedNamespace(FULLY_QUALIFIED_NAMESPACE)
+            .setPartitionId(PARTITION_ID)
+            .setSequenceNumber((long) 1);
+
+        when(jedisPool.getResource()).thenReturn(jedis);
+        when(jedis.exists(PREFIX)).thenReturn(true);
+
+        StepVerifier.create(store.updateCheckpoint(checkpoint))
+            .expectNextCount(0L)
```

Comment:
Why are we expecting 0L? You can simply use

```
StepVerifier.create(store.updateCheckpoint(checkpoint))
    .verifyComplete();
```

Reply:
That seems a lot easier! I was thinking that since I'm expecting no elements in the Flux I could do it that way too.